pages/linux/unzipsfx.md

```diff
@@ -0,0 +1,24 @@
+# unzipsfx
+
+> Create a self extracting compressed binary file by prepending self extracting stubs on a `zip` file.
+> More information: <https://manned.org/unzipsfx>.
+
+- Create a self extracting binary file of a `zip` archive:
+
+`cat unzipsfx {{archive.zip}} > {{filename}} | chmod 755 {{filename}}`
+
+- Extract a self extracting binary in the current directory:
+
+`{{./path/to/binary)}}`
```

Comment:
Oh! not sure about this because our pages are for examples of a specific command and not about commands related to the first one, e.g. start `mongod` via `systemctl`. @waldyrious wdyt?

Reply:
Oh! In that case, we have only the first command to document. The `unzipsfx` just prepand stubs to an existing zip file. It has no significance after that. @navarroaxel  @waldyrious Let me know what needs to be done in this case.

Reply:
Hmm, it's a tricky case indeed. It's similar to the tldr pages for interactive commands like `vim`, where the commands one uses inside are not prepended by the command name (i.e. they're not command lines inputted in the shell in the traditional sense), but are powered by the command being documented.

Assuming the -t, -c and -z options are provided/enabled by `unzipsfx`, I would be OK with the three examples below, which would mean we it would be inconsistent to omit this one. Can you confirm, @Nikhil-Samant?

In any case, I'd wait for other maintainers to comment on this PR because it is indeed a little unorthodox.